### ISSUE TITLE
RocketChat freezing (javascript aborting) on image preview

### ISSUE THREAD

**Issue by soundstorm** [RocketChat/Rocket.Chat#7224]
Rocket.Chat Version: 0.57.0.rc-1

When clicking an image title, the image will open without problems in a new tab. But the inline preview 
yields `TypeError: undefined is not an object (evaluating 't.replace')` leading to no more execution of anything when clicked:
```js
                                        o.importSync("photoswipe/dist/photoswipe-ui-default", {
                                            default: function(t) {
                                                i = t
                                            }
                                        }, 1), o.importSync("photoswipe/dist/photoswipe.css");
                                        var a = function(t) {
                                            return t.replace(/&/g, "&amp;").replace(/</g, "&lt;").replace(/>/g, "&gt;")
                                        };
```
[photoswipe-js#L5](https://github.com/RocketChat/Rocket.Chat/blob/214a7e2386273c008d71c16cb4ee0c36a1c4a7ad/packages/rocketchat-ui/client/views/app/photoswipe.js#L5)

**Comment by soundstorm** [comment]
Caused by #7121

**Comment by soundstorm** [comment]
Happens (only?) with images from Slackbridge. As far as I can see:
Images uploaded with RC have "data-description" even if empty, that's missing for Slackbridge images (and descriptions are not synced at all).

**Comment by soundstorm** [comment]
When viewing an image uploaded with RC - not sure depending on what, as I didn't upload the file - the same happens when the preview popup opens, so no SlackBridge-only bug.
Not reproducible to me at the moment what caused it with that specific image (tried to upload other image with same title and no description as the image was also downloaded the image and reuploaded it).

**Comment by soundstorm** [comment]
Fixed by #7187
